lambdas/utils/document_type_utils.py

```diff
@@ -0,0 +1,18 @@
+from enums.supported_document_types import SupportedDocumentTypes
+
+
+def doc_type_is_valid(value: str) -> bool:
+    if extract_document_type_to_enum(value) is None:
+        return False
+    return True
+
+
+def extract_document_type_to_enum(value: str) -> list[SupportedDocumentTypes]:
+    doc_type = value.replace(" ", "")
+
+    if doc_type == SupportedDocumentTypes.LG.value:
+        return [SupportedDocumentTypes.LG]
+    elif doc_type == SupportedDocumentTypes.ARF.value:
+        return [SupportedDocumentTypes.ARF]
+    elif set(doc_type.split(",")) == set(SupportedDocumentTypes.list_names()):
```

Comment:
What happens if this condition does not occur? 
What happens if we introduce a third type and introduce the need fopr two types instead of the full three?

Reply:
Can we add a default empty list and add the enum if the type is in the `doc_type` array?

Reply:
That looks like it manages error cases now.